test/kvao/ttl.suite.js

```diff
@@ -14,19 +23,19 @@ module.exports = function(dataSourceFactory, connectorCapabilities) {
     it('gets TTL when key with unexpired TTL exists - Promise API',
     function() {
       return Promise.resolve(
-          CacheItem.set('a-key', 'a-value', {ttl: 1000}))
-        .delay(1)
+          CacheItem.set('a-key', 'a-value', {ttl: INITIAL_TTL}))
+        .delay(Math.max(1, Math.floor(TTL_PRECISION / 10)))
```

Comment:
Great point! As I was writing down the comment, I realised I have a bug in my formula, I'll fix it along the way.